android-components/components/feature/addons/src/main/res/values/strings.xml

```diff
@@ -173,6 +173,8 @@
     <string name="mozac_feature_addons_successfully_installed">Successfully installed %1$s</string>
     <!-- Text shown after failed to install an add-on. %1$s is the add-on name. -->
     <string name="mozac_feature_addons_failed_to_install">Failed to install %1$s</string>
+    <!-- Text shown when attempting to install an add-on and a network error happened. -->
+    <string name="mozac_feature_addons_failed_to_install_network_error">The add-on could not be downloaded because of a connection failure</string>
     <!-- Text shown when attempting to install a blocklisted add-on. %1$s is the add-on name. -->
     <string name="mozac_feature_addons_blocklisted">%1$s could not be installed because it has a high risk of causing stability or security problems</string>
```

Comment:
I tried to install a blocklisted extension from the Fenix manager and I get this error message, which is great, except that the `extension` is `null` (the one coming from GV) so we don't have the add-on name here.

Reply:
Thanks for testing! Could you add the steps to reproduce for me to try to replicate it?

Reply:
STR:

1. checkout https://github.com/Rob--W/android-ext-qa locally
2. go into the checked out directory and run: `adb push ext-qa-data/ /data/local/tmp/ext-qa-data`
3. open Firefox Nightly (Desktop) and connect to Fenix via the debugging protocol
4. run the code snippet in the privileged context:

```
IOUtils.copy(
  "/data/local/tmp/ext-qa-data/mozilla_components_addon_collection_en_Extensions-for-Android.json",
  Services.env.get("HOME") + "/mozilla_components_addon_collection_en_Extensions-for-Android.json"
);
```

Reply:
I think it should be working with the new changes, would you mind re-testing ? :) 
![image](https://github.com/mozilla-mobile/firefox-android/assets/773158/360f817b-63b4-4af9-bb4c-1fb7912a1b6a)